apps/core/models.py

```diff
@@ -2,4 +2,19 @@
 
 
 class Flag(AbstractUserFlag):
-    """ Custom version of waffle feature Flag model goes here """
+    """ Custom version of waffle feature Flag model """
+    """ This makes future extensions nicer """
+    def is_active(self, request):
+        # exract the current app's owner (user)
+        self.app_user = request.auth.application.user
+        return super(Flag, self).is_active(request)
```

Comment:
I'm not seeing this method `.is_active(request)` being used anywhere. Wondering if it should be removed?

Reply:
Oh.  Yeah.  It could be removed.  This is left over from an earlier implementation.  Removing.